torch/lib/THD/CMakeLists.txt

```diff
@@ -45,7 +45,14 @@ FIND_PACKAGE(MPI)
 
 FIND_PACKAGE(Gloo)
 
-FIND_PACKAGE(Caffe2 REQUIRED)
+# I don't know if this branch actually still matters, so I am leaving
+# it in. In particular, there is a neighboring file called build.sh
```

Comment:
It was used for development. But considering THD is deprecated and soon to disappear, I doubt it's needed.
I'd lean towards removing it. cc: @teng-li to double-check